Lib/fontbakery/profiles/shared_conditions.py

```diff
@@ -396,17 +396,23 @@ def get_instance_axis_value(ttFont, instance_name, axis_tag):
 
 @condition
 def regular_wght_coord(ttFont):
-    return get_instance_axis_value(ttFont, "Regular", "wght")
+    upright = get_instance_axis_value(ttFont, "Regular", "wght")
+    italic = get_instance_axis_value(ttFont, "Italic", "wght")
+    return upright if upright is not None else italic
```

Comment:
Note: you cannot simply do `upright or italic` since `0 or None` will return None in Python.

Reply:
Can you add this note as a comment in the source, please?